lib/response.js

```diff
@@ -61,12 +61,12 @@ module.exports = res
 
 res.status = function status(code) {
   // Check if the status code is not an integer
-  if (!Number.isInteger(code)) {
-    throw new TypeError(`Invalid status code: ${JSON.stringify(code)}. Status code must be an integer.`);
+  if (typeof code !== 'number' || !Number.isInteger(code)) {
```

Comment:
`Number.isInteger()` checks if the argument is a number - the `typeof` check is unnecessary.

https://262.ecma-international.org/15.0/index.html#sec-number.isinteger
https://262.ecma-international.org/15.0/index.html#sec-isintegralnumber